tests/acceptance/tests/Visual/Administration/Promotions.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect, setViewport, replaceElements } from '@fixtures/AcceptanceTest';
+
+test('Visual: Promotions Listing Page', { tag: '@Visual' }, async ({ 
+    ShopAdmin, 
+    TestDataService, 
+    AdminPromotionsListing,
+ }) => {
+    await test.step('Creates a screenshot of the promotions listing page in its empty state.', async () => { 
+        await ShopAdmin.goesTo(AdminPromotionsListing.url());
+        await setViewport(AdminPromotionsListing.page, {
+            width: 1440,
+            contentHeight: 1200,
```

Comment:
The default height I built into the function is 1080. I think that should be enough here, too.